website/settings/defaults.py

```diff
@@ -330,3 +330,5 @@ def parent_dir(path):
 assert (DRAFT_REGISTRATION_APPROVAL_PERIOD > EMBARGO_END_DATE_MIN), 'The draft registration approval period should be more than the minimum embargo end date.'
 
 PREREG_ADMIN_TAG = "prereg_admin"
+
+YO_ACCESS_TOKEN = 'change_me!'
```

Comment:
I am new to this whole YR (Yode Review) thing, but I notice there is no YO coming from my phone.  Might need more testing. . . . .